attesters/sev-snp/collect_evidence.c

```diff
@@ -80,6 +84,89 @@ static int snp_get_report(const uint8_t *data, size_t data_size, snp_attestation
 	return -1;
 }
 
+static size_t curl_writefunc_callback(void *contents, size_t size, size_t nmemb, void *userp)
+{
+  size_t realsize = size * nmemb;
+  snp_attestation_evidence_t *snp_report = (snp_attestation_evidence_t *)userp;
+ 
+ if (snp_report->vcek_len + realsize > VECK_MAX_SIZE){
+    RATS_ERR("vcek size is larger than %d bytes.",VECK_MAX_SIZE);
+    return 0;
+ }
+  memcpy(&(snp_report->vcek[snp_report->vcek_len]), contents, realsize);
```

Comment:
Hi, please refer to [guide](https://github.com/inclavare-containers/inclavare-containers/blob/master/CONTRIBUTING.md#format-c-codes) to use clang-format to format the code style.